lib/rules/lines-around-comment.js

```diff
@@ -131,8 +123,26 @@ module.exports = {
             comments = sourceCode.getAllComments(),
             commentLines = getCommentLineNums(comments),
             emptyLines = getEmptyLineNums(lines),
-            commentAndEmptyLines = commentLines.concat(emptyLines),
-            reportedLines = [];
+            commentAndEmptyLines = commentLines.concat(emptyLines);
+
+        /**
+         * Returns whether or not a token is a comment node type
+         * @param {Token} token The token to check
+         * @returns {boolean} True if the token is a comment node
+         */
+        function isCommentNodeType(token) {
+            return token && (token.type === "Block" || token.type === "Line");
+        }
+
+        /**
+         * Returns whether or not two tokens or nodes are on the same line.
+         * @param {ASTNode|Token} first The first node or token
+         * @param {ASTNode|Token} second The second node or token
+         * @returns {boolean} True if the two tokens or nodes are on the same line.
+         */
+        function areOnSameLine(first, second) {
```

Comment:
Thanks, PR updated. Build is failing for an unrelated issue: https://github.com/eslint/eslint/issues/6096